packages/core/src/modules/connections/ConnectionsModuleConfig.ts

```diff
@@ -13,15 +15,26 @@ export interface ConnectionsModuleConfigOptions {
    * @default false
    */
   autoAcceptConnections?: boolean
+
+  /**
+   * Peer did num algo to use in requests for DID exchange protocol (RFC 0023). It will be also used by default
+   * in responses in case that the request does not use a peer did.
+   *
+   * @default PeerDidNumAlgo.GenesisDoc
```

Comment:
I think originally the idea for the Step 1 of [RFC 0793](https://github.com/hyperledger/aries-rfcs/tree/main/features/0793-unqualfied-dids-transition) was to simply support receiving requests using the new did:peer:4, and keep behaving as before for requests, so I was thinking on keeping the default to 1 in 0.5 and update to 4 in 0.6.

But I'm fine to defaulting it to 4 right now if you prefer so. It will be a breaking change, but there will be plenty in this release.

Reply:
No that makes sense, let's keep it at 1 👍